WcaOnRails/spec/factories/competitions.rb

```diff
@@ -49,7 +49,7 @@
     venueAddress { "My backyard street" }
     external_website { "https://www.worldcubeassociation.org" }
     showAtAll { false }
-    isConfirmed { false }
+    confirmed_at { nil }
```

Comment:
Wouldn't `confirmed_at` be `nil` without stating this explicitly?

Reply:
I think so, but I do slightly prefer being explicit here.